.travis.yml

```diff
@@ -1,4 +1,12 @@
 sudo: false
+env:
+  - CXX=g++-4.8
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    packages:
+    - g++-4.8
```

Comment:
So feel free to add a comment in the travis yml about that, so future contributors might understand what is it about (or even you in the future :D)